contracts/Paper.sol

```diff
@@ -51,13 +58,43 @@ contract Paper {
         require(block.timestamp <= deadlineDate, "The deadline date has passed.");
         _;
     }
+
+    modifier primaryCheckedHasPassed() {
+        require(primaryChecked == true);
+        _;
+    }
+
+    function primaryChecking() public onlyOwner() {
+        primaryChecked = true;
+    }
```

Comment:
If by primary checking you mean validating that the paper is accepted by the editor before the review starts, it should be done in the `PaperManager` (`Journal` may be a better name). There could be a list of proposed papers, and if the editor validates a paper, its author can call a function to start the review process, which creates the `Paper` contract.